### PR TITLE
Schema Response Change

### PR DESCRIPTION
**Pull request by MichaelChung123**
# Description
Removing formSchema from the fetchDrafts response

## Types of changes
1. Made changes to introduce a Boolean schema parameter that includes or removes the formSchema response
2. Added documentation changes
3. Added test changes

## Checklist

- [X] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [X] I have checked that unit tests pass locally with my changes
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by bcgov-nr-csst** [comment]
PR Deployed at: https://chefs-dev.apps.silver.devops.gov.bc.ca/pr-207

**Review by TimCsaky — APPROVED** [review]
looks good

File: app/frontend/src/services/formService.js [hunk 1]
```diff
@@ -87,6 +87,7 @@ export default {
    * @function listDrafts
    * Get any drafts for a form
    * @param {string} formId The form uuid
+   * @param {boolean} [schema=undefined] Require schema in the response
```
Comment (loneil) [inline app/frontend/src/services/formService.js:90]: Not needed

File: app/src/docs/v1.api-spec.yaml [hunk 2]
```diff
@@ -376,6 +376,9 @@ paths:
         - Draft
       parameters:
         - $ref: "#/components/parameters/formIdParam"
+
+# CHANGES
+
```
Comment (loneil) [inline app/src/docs/v1.api-spec.yaml:381]: Extra comment?

File: app/frontend/tests/unit/store/modules/form.actions.spec.js [hunk 3]
```diff
@@ -88,7 +88,11 @@ describe('form actions', () => {
       await store.actions.fetchDrafts(mockStore, 'dId');
 
       expect(mockStore.commit).toHaveBeenCalledTimes(1);
-      expect(mockStore.commit).toHaveBeenCalledWith('SET_DRAFTS', expect.any(Array));
+      expect(mockStore.commit).toHaveBeenCalledWith('SET_DRAFTS', expect.not.arrayContaining([
+        expect.objectContaining({
+          schema: {}
+        })
+      ]));
```
Comment (loneil) [inline app/frontend/tests/unit/store/modules/form.actions.spec.js:95]: Don't think the test needs to know about `schema` (probably no FE changes needed now)

File: app/frontend/src/components/forms/manage/CurrentDraft.vue [hunk 4]
```diff
@@ -92,6 +92,7 @@
 </template>
```
Comment (loneil) [inline app/frontend/src/components/forms/manage/CurrentDraft.vue:92]: We can delete this file

**Review by loneil — CHANGES_REQUESTED** [review]

**Comment by MichaelChung123** [comment]
Rebased

**Comment by bcgov-nr-csst** [comment]
PR Deployed at: https://chefs-dev.apps.silver.devops.gov.bc.ca/pr-207

**Review by loneil — APPROVED** [review]
Looks great, tested out end-to-end form creation and versioning/publishing etc and things are working good.

**Review by TimCsaky — APPROVED** [review]

**Review by jujaga — APPROVED** [review]

**Pull request merged**